cclib/bridge/cclib2pyscf.py

```diff
@@ -22,12 +24,55 @@ def _check_pyscf(found_pyscf):
 def makepyscf(atomcoords, atomnos, charge=0, mult=1):
     """Create a Pyscf Molecule."""
     _check_pyscf(_found_pyscf)
-    mol = gto.Mole(
-        atom = [['{}'.format(atomnos[i]),atomcoords[i]] for i in range(len(atomcoords))],
-        unit="Angstrom",
-        charge=charge,
-        multiplicity=mult
-    )
-    return  mol
-  
+    # See what can be parsed:
+    inputattrs = data.__dict__
+
+    if hasattr(data, "atomcoords") and hasattr(data, "atomnos"):
+        mol = gto.Mole(
+            atom=[
+                ["{}".format(data.atomnos[i]), atomcoords[i]]
+                for i in range(len(atomcoords))
+            ],
+            unit="Angstrom",
+            charge=charge,
+            multiplicity=mult,
+        )
+
+    return mol
+
+
+def makepyscf_from_molden(data):
+    """
+    Creates an molecule object and additional pyscf data formatted for PySCF.
+    Parameters:
+    ----
+    data: cclib data object from parsed output file
+
+    Returns:
+    ----
+    mol: a pyscf molecule object
+        a molecule object that consists of available data in Molden file.
+    pyscf_data: dictionary
+        a dictionary of additional values returned in an appropriate format for use in PySCF. 
+        If available in ccdata, the following objects are available in the dictionary, otherwise, values are None
+    """
+    _check_pyscf(_found_pyscf)
+    temp = tempfile.NamedTemporaryFile(mode="w+t")
+    pyscf_data = {}
+    # taking advantage of pyscf molden parser as it accounts for basis normalization specifications.
+    mldn_obj = ccwrite(data, outputtype="molden", returnstr=True)
+    temp.write(mldn_obj)
+    temp.seek(0)
+    (
+        mol,
+        pyscf_data["mo_energy"],
+        pyscf_data["mo_coeff"],
+        pyscf_data["mo_occ"],
+        pyscf_data["irrep_labels"],
+        pyscf_data["spins"],
+    ) = tools.molden.load(temp.name)

```

Comment:
I enjoy this hack. I don't think we should test PySCF heavily, but can you add a small test to check the contents of `pyscf_data` members against the cclib equivalents?

Reply:
yes! That's no problem, just wanted to make sure the hack was acceptable before writing the tests. Should have these done shortly. Thanks for the feedback.